CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 - Renamed `options.Options` to `options._Options` to make it private.
   [#74](https://github.com/signalfx/splunk-otel-python/pull/74)
 
+- Renamed `splk-py-trace` and `splk-py-trace-bootstrap` to `splunk-py-trace` and `splunk-py-trace-bootstrap` respectively.
```

Comment:
Consider splitting this into two "lines".
One for "added" and the second for "deprecated", because I think that the old one should be still working.

Reply:
Removed from this PR entirely.